### ISSUE TITLE
Runtime Error 7018

### ISSUE THREAD

**Issue by Duchenkuke** [TheGameCreators/GameGuruRepo#3775]
![Unbenannt](https://user-images.githubusercontent.com/35990507/195952683-cca798ba-e4cb-4316-8e11-b54b75e35280.JPG)

Spent the whole day working on my game. I dont know what it is but today, I got more crashes and bugs than the last half year of using gg. I messed up my standalone also and now I get this runtime error. There is nothing useful on the internet on what this means.

Please, give me a hand here.

**Comment by davetgc** [comment]
Hi,

Can you using custom media or older Classic media?

Can you also please give more details on what "messed up", 

Many thanks.

**Comment by Duchenkuke** [comment]
I am using a lot of custom media, yea. 
I had to copy files from gg to the standalone folder in order to get the new level to run.
And that was done, this error message appeared. 

-- But I restored a backup now and will start again from that one. When I get this error again I will let you know.

**Comment by AmenMoses** [comment]
7018 is RUNTIMEERROR_B3DOBJECTLOADFAILED

Usually means that a dbo file is missing.

It would help a lot of the engine actually included the filename in the error report!

**Comment by Duchenkuke** [comment]
> 7018 is RUNTIMEERROR_B3DOBJECTLOADFAILED
> 
> Usually means that a dbo file is missing.
> 
> It would help a lot of the engine actually included the filename in the error report!

THAT! 

If this would be added, it would be very helpful. 
I was able to fix my issue by updating my gg installation but it ran fine for months and the error just suddenly appeared. Very strange. Also I was getting this error right when I started the game and was loading the main menu. So not sure if its only a missing asset. 

Technically I could close this issue now but if we could somehow have better and more detailed error reports - that would be great. Cheers.
